api/app/Providers/AuthServiceProvider.php

```diff
@@ -34,6 +45,40 @@ public function boot()
             if ($request->input('api_token')) {
                 return User::where('api_token', $request->input('api_token'))->first();
             }
+            if ($request->bearerToken()) {
+                $bearerToken = $request->bearerToken(); // 1. extract JWT access token from request.
+                if ($bearerToken) {
+                    $config = Configuration::forAsymmetricSigner(
+                        new Signer\Rsa\Sha256(),
+                        InMemory::empty(),
```

Comment:
Might be worth adding a comment explaining that the private key is only used for generating tokens, which we don't do here